### PR TITLE
feat: build release versions

### PR DESCRIPTION
**Pull request by TomMcL**
### Description

Adding the capability to build and release versions of Vega market sim to PyPI 

### Testing

Build and released packages to PyPITest from branch

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -132,3 +131,14 @@ ComprehensiveMarket(
...
+
+## Release Process
+
+Releases are aligned to Vega core [releases](https://github.com/vegaprotocol/vega/releases/), to provide a snapshot of working code for each release cutoff. To build and release a Vega Market Sim package, follow these steps:
+
+ - Check [releases](https://github.com/vegaprotocol/vega/releases/) for the release tag you wish to target
+ - Update `.env`'s `VEGA_SIM_VEGA_TAG` to reflect the last commit within that release
+ - Update the `version` parameter in `pyproject.toml` to match the tagged Vega Core release
+ - Run `make` followed by a full integration test run
+ - Create a commit containing these changes, then a pull request into `develop`
```
Comment (davidsiska-vega) [inline README.md:143]: I wouldn't actually know what to do at this step.

Comment (davidsiska-vega) [inline README.md]:
```suggestion
 - Create a branch containing these changes, then a pull request into `develop`
```

Comment (TomMcL) [reply README.md]: 👍

**Review by davidsiska-vega — APPROVED** [review]
LGTM with the proviso that I don't really know any of this stuff - but it sounds plausible.

File: poetry.lock [hunk 2]
```diff
@@ -1628,15 +1697,22 @@ python-versions = ">=3.7"
 
 [[package]]
 name = "torch"
-version = "1.12.1"
+version = "1.13.0"
```
Comment (cdummett) [inline poetry.lock:1700]: @TomMcL Here!

**Pull request merged**